src/main/java/tools/map/making/PlacementPicker.java

```diff
@@ -264,23 +264,23 @@ public PlacementPicker(final String mapName) {
       try (InputStream is = new FileInputStream(file.getPath())) {
         System.out.println("Polygons : " + file.getPath());
         polygons = PointFileReaderWriter.readOneToManyPolygons(is);
-      } catch (final IOException ex1) {
-        ex1.printStackTrace();
+      } catch (final IOException e) {
+        ClientLogger.logQuietly("Failed to load polygons: " + file.getAbsolutePath(), e);
```

Comment:
Here is the behaviour changed completely, previously the error was logged to stderr, now it's logged to stdout